### PR TITLE
Improved logging

### PR DESCRIPTION
**Pull request by iloveitaly**
- Removing old requirements file
- Easy log level customization & log queue path
- Some simple debugging notes

### CODE REVIEW

File: aw_client/client.py [hunk 1]
```diff
@@ -17,11 +17,14 @@
...
-logging.getLogger("requests").setLevel(logging.WARNING)
 logger = logging.getLogger(__name__)
-
+print(__name__)
+# run with LOG_LEVEL=DEBUG to customize log level
+log_level = os.environ.get('LOG_LEVEL')
+if log_level:
+    logger.setLevel(logging.__getattribute__(log_level.upper()))
+else:
+    logger.setLevel(logging.WARN)
```
Comment (johan-bjareholt) [inline aw_client/client.py:27]: I think by default we want INFO to be printed and only have WARN on requests.

**Review by johan-bjareholt — COMMENTED** [review]
We don't have any log system like this in any of our other services in activitywatch, if we want to start using something like this we probably want it across the board.

@ErikBjare What do you think?

**Comment by ErikBjare** [comment]
Setting logging level by env variable probably belongs in aw-core since that's where logging config is generally done.

**Comment by iloveitaly** [comment]
> that's where logging config is generally done

This completely makes sense to me, but I was under the assumption that `logging.getLogger(__name__)` pulls a unique logger configuration with the default log level. Does `logging.setLevel` configure a global log level for all loggers in python?

**Comment by iloveitaly** [comment]
@ErikBjare Created a PR to add this to core: https://github.com/ActivityWatch/aw-core/pull/103.

I amended this PR to remove the `setLevel` call directly here, but kept the other misc tweaks.

**Comment by ErikBjare** [comment]
> Does `logging.setLevel` configure a global log level for all loggers in python?

Yes, it sets it for the root logger and will apply to all loggers unless overridden.

File: aw_client/client.py [hunk 2]
```diff
@@ -17,12 +17,8 @@
 from .config import load_config
 from .singleinstance import SingleInstance
 
-
-# FIXME: This line is probably badly placed
-logging.getLogger("requests").setLevel(logging.WARNING)
 logger = logging.getLogger(__name__)
```
Comment (ErikBjare) [inline aw_client/client.py:20]: Readd the logging config for requests:

```suggestion

# FIXME: This line is probably badly placed
logging.getLogger("requests").setLevel(logging.WARNING)
logger = logging.getLogger(__name__)
```

**Comment by ErikBjare** [comment]
Merged this, but https://github.com/ActivityWatch/aw-core/pull/103 also needs to be merged for the instructions re log level to be valid.

**Pull request merged**